core/src/main/java/com/yahoo/ycsb/generator/UniformGenerator.java

```diff
@@ -48,7 +48,7 @@ public UniformGenerator(Collection<String> values)
 	@Override
   public String nextValue()
 	{
-		_laststring = _values.get(_gen.nextValue());
+		_laststring = _values.get(_gen.nextValue().intValue());
```

Comment:
The size of ArrayList is int, so limiting the value.

Also UniformGenerator doesn't seem to be used so maybe we can consider removing.